module/proto/gravity/v1/genesis.proto

```diff
@@ -72,47 +70,56 @@ option go_package = "github.com/cosmos/gravity-bridge/module/x/gravity/types";
 message Params {
   option (gogoproto.stringer) = false;
 
-  string gravity_id                  = 1;
-  string contract_source_hash        = 2;
-  string bridge_ethereum_address     = 4;
-  uint64 bridge_chain_id             = 5;
-  uint64 signed_valsets_window       = 6;
-  uint64 signed_batches_window       = 7;
-  uint64 signed_claims_window        = 8;
-  uint64 target_batch_timeout        = 10;
-  uint64 average_block_time          = 11;
-  uint64 average_ethereum_block_time = 12;
-  bytes  slash_fraction_valset       = 13 [
+  string gravity_id                      = 1;
```

Comment:
So I think this shouldn't be a param because afaik it's not meant to be changed?

Reply:
Do you think this should just be stored in genesis state?

Reply:
yes, that's what I did on the old refactor